deployment/samples/akri-anomaly-detection-app.yaml

```diff
@@ -16,6 +16,19 @@ spec:
       - name: akri-anomaly-detection-app
         image: ghcr.io/project-akri/akri/anomaly-detection-app:latest-dev
         imagePullPolicy: Always
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true
+          readOnlyRootFilesystem: true
+          capabilities:
+            drop: ["ALL"]
+        resources:
```

Comment:
Have we tested that these levels of resources are sufficient for the anomaly app? I think it would be better to leave resource limits out for our samples as they are not meant to be used in production.

Reply:
thanks, removed the resource limits for anomaly detection app